EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,8 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||chaussew.net^
+||louloapi.com^
```

Comment:
Your pr https://github.com/easylist/easylist/pull/8918 was merged, so it's not necessary. Easylist is part of the AdGuard Base in our products.
I also removed clickunder directly.